src/Symfony/Component/HttpFoundation/ServerEvent.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpFoundation;
+
+/**
+ * A server event to send as part of the SSE streaming technique.
+ *
+ * @author Yonel Ceruto <open@yceruto.dev>
+ */
+class ServerEvent
```

Comment:
```suggestion
class ServerSideEvent
```

ServerSideEvent makes the intent clear that it represents an SSE payload.

Reply:
thanks for the suggestion! I chose "ServerEvent" since SSE stands for "Server-Sent Event" and "ServerSideEvent" might be confused with an event that only occurs internally on the server. I updated the class description to make it clearer that it's an event generated on the server intended for streaming to the client, which then becomes a "Server-Sent Event"